app/controllers/messages_controller.rb

```diff
@@ -2,84 +2,94 @@ class MessagesController < ApplicationController
-	  end
+    end
+    respond_to do |format|
+      format.html
+      format.xml { render :xml => @message }
+    end
   end
 
   def create
-	  @message = Message.new(params[:message])
-	  @message.user_id = current_user.id
-	  @message.user_has_seen = true
-	  @message.from_id = current_user.id
-      @message.sent = true
+    @message = Message.new(message_params)
+    @message.user_id = current_user.id
+    @message.user_has_seen = true
+    @message.from_id = current_user.id
+    @message.sent = true
 
-	  if params[:user] != nil
-		  @message.to_id = params[:user][:id]
-	  end
+    if params[:user] != nil
+      @message.to_id = params[:user][:id]
+    end
 
-	  if @message.save and @message.send_message(@message.from_id, @message.to_id)
-		  flash[:notice] = "Message sent!"
-		  redirect_to "/users/" + current_user.id.to_s + "#messages"
-	  else
-		  respond_to do |format|
-			  format.html { render :action => "new" }
-			  format.xml { render :xml => @message.errors, :status => :unprocessable_entity }
-		  end
-	  end
+    if @message.save and @message.send_message(@message.from_id, @message.to_id)
+      flash[:notice] = "Message sent!"
+      redirect_to "/users/" + current_user.id.to_s + "#messages"
+    else
+      respond_to do |format|
+        format.html { render :action => "new" }
+        format.xml { render :xml => @message.errors, :status => :unprocessable_entity }
+      end
+    end
   end
 
-    def show
+  def show
     @message = Message.find_by_id(params[:id])
     if User.find_by_id(@message.from_id) != nil
-        @from = User.find_by_id(@message.from_id)
+      @from = User.find_by_id(@message.from_id)
     else
-        @from = "Deleted User"
+      @from = "Deleted User"
     end
-    
+
     if User.find_by_id(@message.to_id) != nil 
-        @to = User.find_by_id(@message.to_id)
+      @to = User.find_by_id(@message.to_id)
     else
-        @to = "Deleted User"
+      @to = "Deleted User"
     end
     @message.update_attributes :user_has_seen => true
   end
 
   def destroy
-	  message = Message.find_by_id(params[:id])
-	  if message.destroy
-		  flash[:notice] = "Message deleted."
-		  redirect_to "/users/" + current_user.id.to_s + "#messages"
-	  end
+    message = Message.find_by_id(params[:id])
+    if message.destroy
+      flash[:notice] = "Message deleted."
+      redirect_to "/users/" + current_user.id.to_s + "#messages"
+    end
   end
 
+  private
+
   def require_owner
     unless current_user.id == Message.find_by_id(params[:id]).user_id
       store_location
-		  if current_user
-		    flash[:warning] = "Oops! Thats none of your business"
-		    redirect_to :controller => "users", :action => "show", :id => current_user.id 
-	    else
-	      flash[:notice] = "You need to be logged in"
-	      redirect_to "/signin"
+      if current_user
+        flash[:warning] = "Oops! Thats none of your business"
+        redirect_to :controller => "users", :action => "show", :id => current_user.id 
```

Comment:
Line is too long. [86/80]<br>Use the new Ruby 1.9 hash syntax.<br>Trailing whitespace detected.